### PR TITLE
Sajal/gys-6400-start-working-on-storybook

### PR DESCRIPTION
**Pull request by sajald77**
resolves GYS-6400

### CODE REVIEW

File: docs/pr_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+_Short description of the behaviour that was added or modified._
+
+Motivation and Context
+---
+_Information about the wider context and future plans of the changes._
+
+Link to Related Ticket
+---
+_If the task has e.g. a JIRA ticket linked to it, this is the place to add it._
```
Comment (steliosrammos) [inline docs/pr_template.md:14]: Jira -> Linear, although the Linear is not visible to OS contributors. Does it make sense to have this section here>

File: package.json [hunk 2]
```diff
@@ -22,7 +22,9 @@
     "graph:generate:staging": "yarn graph:generate staging",
     "graph:generate:development": "run(){ yarn graph:fetch:development && graphql-codegen --config codegen.ts && rm schema.graphql && yarn graph:format; }; run",
     "caddy:trust-ca": "bash ./scripts/trust-caddy-cert.sh",
-    "changelog": "yarn conventional-changelog -p conventionalcommits -i CHANGELOG.md -s"
+    "changelog": "yarn conventional-changelog -p conventionalcommits -i CHANGELOG.md -s",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
```
Comment (steliosrammos) [inline package.json:27]: Bit of a nitpick: can we change these to ``storybook:dev` and `storybook:build` to keep a similar format as the other commands.

File: src/stories/components/input/Button.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { BellIcon } from '@chakra-ui/icons'
+import { Button as ChakraButton } from '@chakra-ui/react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
```
Comment (steliosrammos) [inline src/stories/components/input/Button.stories.tsx:5]: Let's add a section about this to the CONTRIBUTE.md, or whichever doc makes most sense.

File: src/stories/components/input/Button.stories.tsx
Comment (steliosrammos) [inline src/stories/components/input/Button.stories.tsx:1]: Do stories replace component files?

**Review by steliosrammos — CHANGES_REQUESTED** [review]
Excited to have Storybook! The PR looks good, just added some small comments before we merge this.

File: src/stories/components/input/Button.stories.tsx
Comment (sajald77) [reply src/stories/components/input/Button.stories.tsx:1]: No they do not. They are not even used inside the whole app, I think we can move the stories outside the src files even, like we have the test files for cypress.

File: docs/pr_template.md (hunk 1 above)
Comment (sajald77) [reply docs/pr_template.md:14]: Yes, but I haven't changed this file, I think I moved it from one location to another. Will update it though.

File: package.json (hunk 2 above)
Comment (sajald77) [reply package.json:27]: I was wondering if I should do that or not too, thanks, will update it.

File: src/stories/components/input/Button.stories.tsx (hunk 3 above)
Comment (sajald77) [reply src/stories/components/input/Button.stories.tsx]: Adding it to code_guidelines makes more sense, added it there.

**Pull request merged**